bin/server

```diff
@@ -43,6 +43,10 @@ authConfig.scm = scm;
 const ArtifactPlugin = require('screwdriver-artifact-bookend');
 const artifact = new ArtifactPlugin(ecosystem.store);
 
+// Build cache plugin
+const CachePlugin = require('screwdriver-cache-bookend');
+const cache = new CachePlugin(ecosystem.api);
```

Comment:
There is no `ecosystem.api`, do you mean `httpd.uri`

Reply:
This one? https://github.com/screwdriver-cd/screwdriver/blob/e0e2b8a998abc949585fee4f3f073f181bf80c83/bin/server#L24